doc/manual/expressions/language-operators.xml

```diff
@@ -32,7 +32,9 @@ weakest binding).</para>
         <replaceable>e</replaceable>.  (An attribute path is a
         dot-separated list of attribute names.)  If the attribute
         doesn’t exist, return <replaceable>def</replaceable> if
-        provided, otherwise abort evaluation.</entry>
+        provided, otherwise abort evaluation. A function equivalent to this
+        operator, <literal>getAttrFromPath</literal>, is available in the
+        <literal>nixpkgs</literal> library.</entry>
```

Comment:
May also be worth mentioning `builtins.getAttr` and `builtins.hasAttr` respectively.

Reply:
I agree with @ericsagnes for `hasAttrByPath`, but not for `getAttrFromPath`

```
nix-repl> let s = { a = { b = 1; }; }; in  (builtins.getAttr "a" s).b                                                                                                                                           
1 
nix-repl> let s = { a = { b = 1; }; }; in  (s.a).b                                                                                                                                                              
1                                                                                                                                                                                                               

nix-repl> let s = { a = { b = 1; }; }; in  s.(a.b)                                                                                                                                                              
error: syntax error, unexpected '(', expecting ID or OR_KW or DOLLAR_CURLY or '"', at (string):1:36                                                                                                             
```